### PR TITLE
Updated documentation with latest R2V info

### PR DESCRIPTION
**Pull request by bradleynull**
## Overview

This brings the R2V RTD page up to the latest hardware and software documentation. This includes removing the Xsens as the recommended GNSS and adding the VN-310, updating the data collection page with latest info, and adding the data products page. 

## Changes
* Data Collection 
  * Updated to remove Xsens replace with VN
  * Added new ladybug imagery
  * Change GNSS calibration procedures for simpler method with VN
  * Updated data collection screenshot
* Data Processing 
  * Now supported in Ubuntu 20.04
  * Changed from repo access to deployment to using Docker images with scripts
  * Updated with latest calibration tools and scripts that run them
* Data Products 
  * Updated the GNSS data playback video
  * Updated SLAM text
  * Added full color cloud
  * Added static object detection

## Testing

Check out [this branch](https://monodrive.readthedocs.io/en/r2v_updates_april2021/)

### CODE REVIEW

**Review by katiemacaulay — COMMENTED** [review]
@bradleynull, this looks great! Just a few suggestions/comments.

**Review by citlaligm — CHANGES_REQUESTED** [review]
Looking good Brad! I didn't catch any other issues than the ones Katie pointed out.

File: docs/r2v/data_collection_quick_start.md [hunk 1]
```diff
@@ -68,34 +66,31 @@ monoDrive vehicle the JSON should look like:
...
-1. Once the IMU yaw and the Estimated Yaw are relatively close (~1-2 degrees), data collection can begin
-1. To begin recording data, click "Record"
-1. When data collection is complete, click "Stop"
+1. Ensure that the JSON configurations are correct in the "Configure" tab.
+1. With the "Control" tab selected, click the "Run" arrow on the top-left of 
+the monoDrive LabVIEW data logger application. The data logger will connect to 
+all sensors and ensure they are functioning as expected.
+1. The software is now connected and can display sensor outputs. 
+To begin recording data, click "Record".
+1. When data collection is complete, click "Stop Rec"
```
[the remark this answers is not in the document]
Comment (bradleynull) [reply docs/r2v/data_collection_quick_start.md:91]: Fixed.
.

File: docs/r2v/data_collection_quick_start.md [hunk 2]
```diff
@@ -120,30 +117,19 @@ in subfolders that match the year, date, and time by hour. To verify that a run
...
-<div class="img_container">
-    <img class='lg_img' src="../imgs/google_earth_gnss_output.png"/>
-</div>
-
 ## Parsing Data Into KITTI Format
 
 For use in the monoDrive Real-to-Virtual data processing pipeline, the collected
 data needs to be converted to the 
-[KITTI format](https://www.cvlibs.net/datasets/kitti/index.php). The `monodrive_to_kitti.py` script in the monoDrive Mapping Toolkit allows users
+[KITTI format](https://www.cvlibs.net/datasets/kitti/index.php). The
```
[the remark this answers is not in the document]
Comment (bradleynull) [reply docs/r2v/data_collection_quick_start.md:131]: Updated link.

File: docs/r2v/data_processing_quick_start.md [hunk 3]
```diff
@@ -9,38 +9,25 @@ levels usable in the monoDrive Simulator.
...
 * [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-docker#ubuntu-16041804-debian-jessiestretchbuster)
 
 ### Processing Pipeline Installation
 
 Users of the Real-to-Virtual software will have access to the monoDrive 
-Real-to-Virtual deployment repository. This repo contains all software for 
-processing real-to-virtual data into assets for us in Unreal Engine and the
-monoDrive Simulator.
+Real-to-Virtual deployment. These binary Docker images and configuration files  
+contain all software for processing real-to-virtual data into assets for use in
```
[the remark this answers is not in the document]
Comment (bradleynull) [reply docs/r2v/data_processing_quick_start.md:21]: Fixed.

**Comment by bradleynull** [comment]
@katiemacaulay @citlaligm Thanks for the review! Addressed all your comments and I think this is ready for another look.

**Review by katiemacaulay — APPROVED** [review]
@bradleynull , ready to go!

**Review by citlaligm — APPROVED** [review]
Looks good!

File: docs/r2v/data_processing_quick_start.md [hunk 4]
```diff
@@ -72,12 +74,13 @@ colormap. The initial alignment can be seen in the following picture:
...
-    --lidar_pos_offset 0.0 0.15 0.0 \
-    --lidar_angle_offset -0.5 0.0 0.0
+```
+{
+    "lidar_pos_offset": [0.0, 0.20, -0.06],
+    "lidar_angle_offset": [-0.3, -0.15, 0.0],
+    "gnss_pos_offset": [0.0, 0.0, 0.0],
+    "gnss_angle_offset": [0.0, 0.0, 0.0]
+}
 ```
```
Comment (devinaconley) [inline docs/r2v/data_processing_quick_start.md:85]: Can we rephrase this to make it clear the file is already in the correct location?

>Once the values have been determined, place them in the 
`config/sensor_offset.json` file for use in the pipeline.

File: docs/r2v/data_processing_quick_start.md [hunk 5]
```diff
@@ -141,6 +163,78 @@ objects in the cloud become more apparent:
 Once the values have been determined, place them in the 
 `config/sensor_offset.json` file for use in the pipeline.
```
Comment (devinaconley) [inline docs/r2v/data_processing_quick_start.md:164]: Same comment - make it clear the file is already there now

File: docs/r2v/data_processing_quick_start.md [hunk 6]
```diff
@@ -141,6 +163,78 @@ objects in the cloud become more apparent:
...
 
+## Calibrating Texture Mapping
+
+After the mesh data has been created, the Texture Mapping Alignment Tool can
```
Comment (devinaconley) [inline docs/r2v/data_processing_quick_start.md:168]: Probably want to make this even more clear that the next step "Running the pipeline" needs to at least run through meshing first

File: docs/r2v/data_processing_quick_start.md [hunk 7]
```diff
@@ -141,6 +163,78 @@ objects in the cloud become more apparent:
...
+```
+
+This will bring up the GUI window for manipulating the alignment. The mesh and
+texture mapping are controlled through the mouse and keyboard:
+
+* `Right mouse` + `w`/`s`/`a`/`d`: Contol the camera view of the mesh
+* `4`/`5`: Rotate texture alignment in yaw
+* `6`/`7`: Rotate texture alignment in roll
+* `8`/`9`: Rotate texture alignment in pitch
+
```
Comment (devinaconley) [inline docs/r2v/data_processing_quick_start.md:189]: `n`: forward one image
`m`: back 10 images

**Review by devinaconley — APPROVED** [review]
This looks good! Just left a few minor comments

File: docs/r2v/data_processing_quick_start.md (hunk 4 above)
Comment (bradleynull) [reply docs/r2v/data_processing_quick_start.md:85]: Fixed.

File: docs/r2v/data_processing_quick_start.md (hunk 5 above)
Comment (bradleynull) [reply docs/r2v/data_processing_quick_start.md:164]: Fixed.

File: docs/r2v/data_processing_quick_start.md (hunk 6 above)
Comment (bradleynull) [reply docs/r2v/data_processing_quick_start.md:168]: Updated.

File: docs/r2v/data_processing_quick_start.md (hunk 7 above)
Comment (bradleynull) [reply docs/r2v/data_processing_quick_start.md:189]: Fixed.
